heartbeat/docs/getting-started.asciidoc

```diff
@@ -1,184 +1,73 @@
-
-include::{libbeat-dir}/shared-download-and-install.asciidoc[]
-
-[[deb]]
-*deb:*
-
-ifeval::["{release-state}"=="unreleased"]
-
-Version {version} of {beatname_uc} has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-["source","sh",subs="attributes"]
-----------------------------------------------------------------------
-curl -L -O {downloads}/heartbeat/heartbeat-{version}-amd64.deb
-sudo dpkg -i heartbeat-{version}-amd64.deb
-----------------------------------------------------------------------
-
-endif::[]
-
-[[rpm]]
-*rpm:*
-
-ifeval::["{release-state}"=="unreleased"]
-
-Version {version} of {beatname_uc} has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-["source","sh",subs="attributes"]
-----------------------------------------------------------------------
-curl -L -O {downloads}/heartbeat/heartbeat-{version}-x86_64.rpm
-sudo rpm -vi heartbeat-{version}-x86_64.rpm
-----------------------------------------------------------------------
-
-endif::[]
-
-[[mac]]
-*mac:*
-
-ifeval::["{release-state}"=="unreleased"]
-
-Version {version} of {beatname_uc} has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-["source","sh",subs="attributes"]
-------------------------------------------------
-curl -L -O {downloads}/heartbeat/heartbeat-{version}-darwin-x86_64.tar.gz
-tar xzvf heartbeat-{version}-darwin-x86_64.tar.gz
-------------------------------------------------
-
-endif::[]
-
-include::{libbeat-dir}/shared-brew-install.asciidoc[]
-
-[[linux]]
-*linux:*
-
-ifeval::["{release-state}"=="unreleased"]
-
-Version {version} of {beatname_uc} has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-["source","sh",subs="attributes"]
-------------------------------------------------
-curl -L -O {downloads}/heartbeat/heartbeat-{version}-linux-x86_64.tar.gz
-tar xzvf heartbeat-{version}-linux-x86_64.tar.gz
-------------------------------------------------
-endif::[]
-
-[[docker]]
-*docker:*
-
-See <<running-on-docker, Running on Docker>> for deploying Docker containers.
-
-[[win]]
-*win:*
-
-ifeval::["{release-state}"=="unreleased"]
+To download and install {beatname_uc}, use the commands that work with your
+system:
 
-Version {version} of {beatname_uc} has not yet been released.
+include::{libbeat-dir}/tab-widgets/install-widget.asciidoc[]
 
-endif::[]
+[float]
+[[other-installation-options]]
+==== Other installation options
 
-ifeval::["{release-state}"!="unreleased"]
+* <<setup-repositories,APT or YUM>>
+* https://www.elastic.co/downloads/beats/{beatname_lc}[Download page] 
+* <<running-on-docker,Docker>>
 
-. Download the Heartbeat Windows zip file from the
-https://www.elastic.co/downloads/beats/heartbeat[downloads page].
+[float]
+[[set-connection]]
+=== Step 2: Set the connection information
```

Comment:
What about "Step 2: Connect to the Elastic Stack"

Reply:
fixed.